website/docs/r/integration_pagerduty.html.markdown

```diff
@@ -39,12 +39,63 @@ resource "datadog_integration_pagerduty" "pd" {
 The following arguments are supported:
 
 * `services` - (Optional) Array of PagerDuty service objects.
-  * `service_name` - (Required) Your Service name in PagerDuty.
-  * `service_key` - (Required) Your Service name associated service key in Pagerduty.
+  * `service_name` - (Required) A name for your service to use with Datadog.
+  * `service_key` - (Required) The service's integration key from PagerDuty.
 * `schedules` - (Optional)  Array of your schedule URLs.
 * `subdomain` - (Required) Your PagerDuty account’s personalized subdomain name.
 * `api_token` - (Optional) Your PagerDuty API token.
 
+
+# datadog_integration_pagerduty_service
```

Comment:
This should be moved to its own `website/docs/r/integration_pagerduty_service.html.markdown` file.